platform/viewer/public/config/dicomweb_relative.js

```diff
@@ -0,0 +1,160 @@
+window.config = {
```

Comment:
This is just a generic config that accesses a dicomweb relative directory - it is added for consistency with other builds.  Note that it will get removed eventually when it gets moved into a custom mode handler, but for now it is important to allow this one to exist.